include/BMS/BMS.hpp

```diff
@@ -50,6 +91,95 @@ class BMS {
      */
     DEV::BQ76952 bq;
 
+    /**
+     * The current state of the BMS
+     */
+    State state;
+
+    /**
+     * The interlock which is used to detect a cable plugged in
+     */
+    DEV::Interlock& interlock;
+
+    /**
+     * This GPIO is connected to the ALARM pin of the BQ. The BQ can be
+     * configured to toggle the ALARM pin based on certain safety parameters.
+     * If the alarm pin is in it's active state, should assum it is unsafe
```

Comment:
Woops, good spelling catch. We can safely assume because down the line we configure the BQ to output an alarm under certain conditions, so we can assume that we've configured the BQ to respond to critical issues.